app/routes/admin/users/list.js

```diff
@@ -0,0 +1,98 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    let route = this.get('router.url').split('/');
+    return this.i18n.t(route[route.length - 1]);
```

Comment:
And don't split it from the URL. Not a good practice. Directly use the dynamic segment value.